trading_calendars/calendar_utils.py

```diff
@@ -50,6 +61,16 @@
     'ICEUS': 'IEPA',
     'NYFE': 'IEPA',
     'CFE': 'XCBF',
+    'ENX': 'XPAR',  # Unsure about this
+    'ZZFR': 'XPAR',
+    'COAL': 'XPAR',
+    'LIF': 'XPAR',
+    'ZZBE': 'XBRU',
+    'ZZPT': 'XLIS',
```

Comment:
where are these from? I don't see them grepping for Portugal in the ISO code csv